### DISCUSSION TITLE
compile to a wrong  version

### DISCUSSION THREAD

**Discussion by YZJ** [openssl/openssl#24281] [Q&A]
when I checkout to  branch OpenSSL_1_1_1-stable,  
run .bat

nmake clean
perl Configure VC-WIN64A --debug --prefix=D:\openssl-1.1.1\openssl\out\Lib\x64\Debug --openssldir=D:\openssl-1.1.1\openssl\out\ssl no-shared no-tests no-asm
nmake test
nmake install_sw

finally I get the wrong version of openssl.exe  . 

D:\openssl-1.1.1\openssl\out\Lib\x64\Debug\bin\openssl.exe version
OpenSSL 3.4.0-dev  (Library: OpenSSL 3.4.0-dev ).

help.
what should I do to compile to get the version of 1.1.1 openssl.exe

**Comment by t8m** [comment]
Most likely you've not really checked out the OpenSSL_1_1_1-stable branch.
